Source/ASControlNode.mm

```diff
@@ -230,6 +238,17 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
                           withEvent:event];
 }
 
+- (void)_cancelTrackingForControlEvents:(ASControlNodeEvent)controlEventMask withEvent:(UIEvent *)event
```

Comment:
This should probably be:

```objc
- (void)_cancelTrackingWithEvent:(UIEvent *)event
```
and then use the `ASControlNodeEventTouchCancel` literal directly here.